lib/openshift-rest-client.js

```diff
@@ -144,6 +151,9 @@ async function openshiftClient (settings = {}) {
   }
 
   const client = new Client(clientConfig);
+  if (settings.loadSpecFromCluster) {
+    await client.loadSpec();
```

Comment:
`await` will throw if the promise is rejected. Maybe add try catch here?

Reply:
yea.  i think that makes sense.  Pretty sure that the underlying client fails silently and will always resolve.  I will probably also do the same.   

I do wonder if it makes sense to load the spec we include if this fails.   

The reason for "failing" would be if there was no "/openapi/v2 " or "swagger.json" endpoints or not logged in and the spec couldn't be loaded.  In that case there would be no api generated,   i wonder if that is what we want.

Reply:
Maybe load the included spec, and emit a warning.

Reply:
Yea,  thats what ii was thinking, but forgot to mention :)

Reply:
i think my latest push resolves this